package/yast2-firewall.spec

```diff
@@ -26,8 +26,10 @@ Url:            https://github.com/yast/yast-firewall
 
 Source0:        %{name}-%{version}.tar.bz2
 
-BuildRequires:  perl-XML-Writer update-desktop-files yast2-testsuite
+BuildRequires:  update-desktop-files
 BuildRequires:  yast2-devtools >= 4.2.2
+# for proposing selinux
+BuildRequires:  yast2-security >= 4.2.15
```

Comment:
So, we should merge https://github.com/yast/yast-security/pull/83, right?

Reply:
yes, it is dependency, otherwise test cannot be run